src/devices/sound/upd934g.cpp

```diff
@@ -127,25 +124,24 @@ void upd934g_device::write(offs_t offset, uint8_t data)
 		// format of data written here is:
 		// 76------  command
 		// --5432--  sample number
-		// ------10  volume?
+		// ------10  output channel
 		m_sample = (data >> 2) & 0x0f;
 
 		switch (data >> 6)
 		{
 		case 0:
-			logerror("CMD STORE ADDRESS sample %x\n", m_sample);
+			LOG("CMD STORE ADDRESS sample %x\n", m_sample);
 			break;
-		case 1:
-		case 2:
-		case 3:
-			logerror("CMD PLAY sample %x (channel %d, effect %d)\n", m_sample, m_sample >> 1, data >> 6);
-			if (m_sample < 8)
-			{
-				m_channel[m_sample >> 1].pos = m_addr[m_sample];
-				m_channel[m_sample >> 1].playing = m_sample;
-				m_channel[m_sample >> 1].volume = data & 0x03;
-				m_channel[m_sample >> 1].effect = data >> 6;
-			}
+		case 1: // normal
+		case 2: // muted
+		case 3: // accented
+		{
+			const u8 ch = (data & 3) ^ 2; // effective order seems to be "2, 3, 0, 1"
```

Comment:
Why the `^ 2`? I mean, all outputs go to the same speaker anyway, so the order shouldn't matter.

Reply:
I did that mostly so that the output ordering would still be correct for the RZ-1, since that does have the individual volume control and line outputs for each sample.